app/src/main/java/com/codesoom/assignment/errors/InvalidAccessTokenException.java

```diff
@@ -0,0 +1,8 @@
+package com.codesoom.assignment.errors;
+
+public class InvalidAccessTokenException  extends RuntimeException {
+    public InvalidAccessTokenException(String accessToken)
+    {
+        super("Invalid Access token : " + accessToken);
+    }
```

Comment:
일반적인 Java 줄괄호 컨벤션을 지켜 주세요.
```suggestion
public class InvalidAccessTokenException  extends RuntimeException {
    public InvalidAccessTokenException(String accessToken) {
        super("Invalid Access token : " + accessToken);
    }
```

Reply:
앗 수정했습니다!